packages/stopwatch/package.json

```
@@ -0,0 +1,52 @@
+{
+	"name": "@sapphire/stopwatch",
+	"version": "1.0.0",
+	"description": "A simple Stopwatch class for JavaScript.",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"browser": "dist/index.umd.js",
+	"unpkg": "dist/index.umd.js",
+	"types": "dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"homepage": "https://github.com/sapphire-project/utilities/tree/main/packages/stopwatcj",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
+		"build": "tsc -p .",
```

```suggestion
		"build": "tsc -b src",
		"postbuild": "gen-esm-wrapper dist/index.js dist/index.mjs",
```

Also add a root dev dependency for `gen-esm-wrapper`